protocols/v2/framing-sv2/src/framing.rs

```diff
@@ -44,11 +44,9 @@ impl<T: Serialize + GetSize, B: AsMut<[u8]> + AsRef<[u8]>> From<Sv2Frame<T, B>>
 
 /// Abstraction for a SV2 Frame.
```

Comment:
we should expand this comment to make it clear what each `enum` variant is

something like:
```
/// Both `Payload` and `Raw` variants carry the `Header` as metadata
/// `Payload` means the `Sv2Frame` carries a payload, but it has not yet been serialized
/// `Raw` means the `Sv2Frame` has been fully serialized, where the `serialized` field contains both `header` + `payload`
```